avm/res/batch/batch-account/main.json

```diff
@@ -1486,4 +1486,4 @@
       "value": "[if(and(coalesce(tryGet(parameters('managedIdentities'), 'systemAssigned'), false()), contains(reference('batchAccount', '2022-06-01', 'full').identity, 'principalId')), reference('batchAccount', '2022-06-01', 'full').identity.principalId, '')]"
     }
   }
-}
+}
```

Comment:
There seems to be an unintended change, likely done by auto-formatting.

Reply:
This had been removed in a prior commit, and doesn't look to exist in current PR anymore.